hack/vendor.sh

```diff
@@ -19,7 +19,12 @@ case $# in
 	;;
 # If user passed arguments to the script
 1)
-	eval "$(grep -E "^clone [^ ]+ $1" "$0")"
+	path=$(pwd)"/hack/vendor.sh"
+	if ! cloneGrep="$(grep -E "^clone [^ ]+ $1" $path)"; then
+    	echo >&2 "error: failed to find 'clone ... $1' in $0"
+    	exit 1
```

Comment:
Looks like you've got a mix of tabs and spaces here. :smile: (should be all tabs)

Also, `$path` is the same as `$BASH_SOURCE`, so I think we should just use that (and quote it, in case users have spaces in their path).

ie:
```diff
+	if ! cloneGrep="$(grep -E "^clone [^ ]+ $1" "$BASH_SOURCE")"; then
+		echo >&2 "error: failed to find 'clone ... $1' in $BASH_SOURCE"
+		exit 1
+	fi
+	eval "$cloneGrep"
```